cime_config/config_grids.xml

```diff
@@ -2885,6 +2915,12 @@
       <desc>mpas.gis1to10km is a variable-resolution, from 1- to 10-km, MALI grid of the Greenland Ice Sheet.</desc>
     </domain>
 
+    <domain name="mpas.gis1to10kmR2">
+      <nx>427386</nx>
+      <ny>1</ny>
+      <desc>mpas.gis1to10km is a variable-resolution, from 1- to 10-km, MALI grid of the Greenland Ice Sheet.</desc>
```

Comment:
```suggestion
      <desc>mpas.gis1to10kmR2 is a variable-resolution, from 1- to 10-km, MALI grid of the Greenland Ice Sheet.</desc>
```

Also, if we are keeping both the original and R2 versions for some reason, update the descriptions to indicate what the difference is

Reply:
See response to the above question. I can certainly update the description though. Thanks for flagging that.

Reply:
Chloe and Charlie suggested it might be nice to keep this grid around for a bit for backward compatibility, since it's what they used to spin up the SMB with. I think if we wanted to push to get rid of it they would be ok with that too.

Reply:
For the reasons noted above, I'm going to assume we keep this support in for a bit longer and resolve this discussion thread for now.